### PR TITLE
Engg:: Small corrections to Engg Diff interface documentation

### LINKED ISSUE #17098: Engg:: Eng diff interface docs minor changes
The engineering diffraction interface documentation requires several minor changes to improve its readability such as capitalisation and clarifications.

### PR DESCRIPTION
**Pull request by DavidFair**
Description of work.
Read through and tidied the documentation for the Engineering Diffraction interface
**To test:**
Doc tests should complete successfully 

Fixes #17098 .

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Engineering_Diffraction.rst [hunk 1]
```diff
@@ -113,13 +122,12 @@ Calibration sample #
...
-  "Enable Spectrum-Nos" by default. This option cannot be used together
-  with Spectrum Nos, as they overlap. For the convenience of users, the
-  GUI will only enable Spectrum Numbers and Customise Bank Name text-fields
-  when Bank Name drop down list is set to "Enable Spectrum-Nos"
-
+  option of "North" and "South", which are equivalent to 1 and 2
+  respectively. 
+  Custom bank mappings can be created by setting the Bank Name option
+  to `Use spectrum numbers`. By specifying a range or list of 
+  spectra to use a new custom bank can be created and subsequently named.
```
Comment (shahrozahmed1) [inline docs/source/interfaces/Engineering_Diffraction.rst]: @DavidFair This line can be rephrased maybe?

File: docs/source/interfaces/Engineering_Diffraction.rst [hunk 2]
```diff
@@ -234,31 +246,38 @@ Output
...
+plotting data in three different formats. 
+
+- One Window - Replacing Plots: will replace the previous graph and plot a new graph on top.
+
+- One Window - Waterfall: will plot all the generated focused workspace graphs in one window 
+  which can be useful while comparing various graphs. 
+
+- Multiple Windows - will plot graph in separate windows. 
+
+However, user may also change the Plot Data representation drop-down box while a run is being carried out. This
```
Comment (shahrozahmed1) [inline docs/source/interfaces/Engineering_Diffraction.rst]: I think this option has been disabled now for some reason

File: docs/source/interfaces/Engineering_Diffraction.rst [hunk 3]
```diff
@@ -168,18 +175,23 @@ Three focusing alternatives are provided:
...
-include different banks and/or combinations of spectra (detectors). In
-the firs option, normal focusing, all the selected banks and all the
-spectra present in the input runs are considered. In the second
-alternative, cropped focusing, all the banks are considered in
+include different banks and/or combinations of spectra (detectors). 
+The behavior for each option is as follows:
+
+1. *Normal focusing* - All the selected banks and spectra present
+in the input runs are considered. The output focused workspace will take 
+suffixes such as *_bank_1, _bank_2*, and so on
```
Comment (shahrozahmed1) [inline docs/source/interfaces/Engineering_Diffraction.rst]: use `name` instead of `take` here and below

**Comment by shahrozahmed1** [comment]
@DavidFair other than that everything looks fine, let me know once you have made these changes

File: docs/source/interfaces/Engineering_Diffraction.rst (hunk 2 above)
Comment (shahrozahmed1) [inline docs/source/interfaces/Engineering_Diffraction.rst]: Removing [this line] (https://github.com/mantidproject/mantid/blob/b7be73a2d42de5e9f0a37741f7a9073d77087033/MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp#L633) should should enable it this again, but probably would require you to disable other features within `groupBox_focus_output_options` group box on the focus tab

**Comment by DavidFair** [comment]
@shahrozahmed1 I've done the changes requested if you want to check for anything else

File: MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp [hunk 4]
```diff
@@ -615,7 +615,8 @@ void EnggDiffractionViewQtGUI::newCalibLoaded(const std::string &vanadiumNo,
   }
 }
 
-void EnggDiffractionViewQtGUI::enableCalibrateFocusFitUserActions(bool enable) {
+void EnggDiffractionViewQtGUI::enableCalibrateFocusFitUserActions(
```
Comment (shahrozahmed1) [inline MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp]: Clang format failing due to this probably

**Comment by shahrozahmed1** [comment]
This looks good, tested it locally works. Enables the feature to change plot during focusing process too. Once all builds pass .. :shipit:

**Pull request merged**